### PR TITLE
issue 1214

### PR DESCRIPTION
**Pull request by iguberman**
Added the info from @josevalim response about how to get around this issue.

### CODE REVIEW

File: bonus_guides/EE_custom_primary_key.md [hunk 1]
```diff
@@ -61,14 +61,14 @@ defmodule HelloPhoenix.Repo.Migrations.CreatePlayer do
 end
 ```
 
-Let's move on to `web/models/player.ex` next. We'll need to add a module attribute `@primary_key {:name, :string, []}` describing our primary key as a string. We'll also need to remove the `field :name, :string` line because this is our new primary key. If this seems unusual, recall that the schema doesn't list the `id` field in models where `id` is the primary key.
+Let's move on to `web/models/player.ex` next. We'll need to add a module attribute `@primary_key {:name, :string, []}` describing our primary key as a string. And we need to tell Phoenix how to convert our data structure to an ID that is used in the routes: `@derive {Phoenix.Param, key: :name}` We'll also need to remove the `field :name, :string` line because this is our new primary key. If this seems unusual, recall that the schema doesn't list the `id` field in models where `id` is the primary key.
```
Comment (lancehalvorsen) [inline bonus_guides/EE_custom_primary_key.md]: What would you think about changing "And" to "Next" in the newly added section and then add a period at the end? That would look like this:

Then we need to tell . . . routes: `@derive {Phoenix.Param, key: :name}`.

**Comment by lancehalvorsen** [comment]
Hi @iguberman, thanks for submitting this correction! I just left one teeny comment. When we get that one settled and the commits squashed, we'll be all set.

Thanks again!

**Comment by iguberman** [comment]
Hi Lance,

Please feel free to make whatever changes you feel would make this document
better.  Or I'll address your suggestions later tonight -- whatever is
easier for you.  :)

Thank you very much!
Irina.

**Comment by lancehalvorsen** [comment]
Hi @iguberman,

Thanks! I'll merge this and take care of the changes. Thanks so much once again!

**Comment by iguberman** [comment]
No -- thank YOU! :)

**Pull request merged**